app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingSectorPage.scala.html

```diff
@@ -90,15 +90,22 @@
       ))
     }
 
-    @H1(messages("undertakingSector.title"), Some("govuk-heading-xl govuk-!-margin-bottom-5"))
-    @P(messages("undertakingSector.p1"), None, Some("sector-desc"))
+
 
     @govukRadios(Radios(
       errorMessage = if(form.hasErrors) {Some(ErrorMessage(
         content = Text(messages("undertakingSector.error.required"))
       ))} else None,
       fieldset = Some(Fieldset(
-        describedBy = Some("sector-desc")
+        describedBy = Some("sector-desc"),
+        legend = Some(Legend(
+        content = Text(messages("undertakingSector.title")),
+        classes = "govuk-fieldset__legend--xl break-word",
```

Comment:
please remove `break-word` as it is not required

Reply:
updated